### DISCUSSION TITLE
rclone-mount as Vorta's pre-backup command

### DISCUSSION THREAD

**Discussion by abdeoliveira** [borgbase/vorta#1202] [General]
I would like to rclone-mount my Google Drive locally to be used by Vorta/Borg as a backup destination. I tried to run 
$rclone --config /path/to/rclone/config.file mount remote:dir /path/to/local/dir --daemon 
as a Pre-backup command in Vorta Shell Commands with no luck. It returns a "non-zero exit code." 

Any help will be greatly appreciated!

**Comment by m3nu** [comment]
File access will be really slow. Wouldn't recommend this for anything serious.

**Reply by abdeoliveira** [reply]
@m3nu ,
Thanks for your input. Actually this is the very opposite of my testings. I  did  a backup of a small data sample (2.6 GB) to rclone-mounted Gdrive and it took 4:09 to complete. The exactly same sample was backup to BorgBase (US) taking 5:30 to finish. Considering network fluctuations etc, I would say both are on par.

**Comment by abdeoliveira** [comment]
Just figured out how to make it work (not elegantly thou):
As a Pre-command: ssh localhost 'rclone mount remote: localdir --daemon; sleep 5'
As a Pos-Command: ssh localhost 'umount localdir'

**Reply by m3nu** [reply]
Seems pretty intuitive. Glad this works as expected.
